docs/references/astro/session-list-store.mdx

```diff
@@ -11,22 +11,59 @@ The `$sessionListStore` store returns an array of [`Session`](https://clerk.com/
 
 The following example demonstrates how to use the `$sessionListStore` store to display the expiration time of the first session in the list.
 
-```tsx {{ prettier: false, filename: 'session-list.tsx' }}
-import { useStore } from '@nanostores/react';
-import { $sessionList } from '@clerk/astro/client';
+<CodeBlockTabs options={['React', 'Vue', 'Svelte']}>
+  ```tsx {{ prettier: false, filename: 'session-list.tsx' }}
+  import { useStore } from '@nanostores/react';
+  import { $sessionList } from '@clerk/astro/client';
 
-export default function Home() {
-  const sessions = useStore($sessionList);
+  export default function Home() {
+    const sessions = useStore($sessionList);
+
+    if (sessions === undefined) {
+      // Handle loading state
+      return null;
+    }
 
-  if (sessions === undefined) {
-    // handle loading state
-    return null;
+    return (
+      <div>
+        <p>Your current session expires at {sessions[0].expireAt}.</p>
+      </div>
+    )
   }
+  ```
+
+  ```vue {{ prettier: false, filename: 'session-list.vue' }}
+  <script setup>
+  import { useStore } from '@nanostores/vue';
+  import { $sessionList } from '@clerk/astro/client';
+
+  const sessions = useStore($sessionList);
+  </script>
+
+  <template>
+    <div v-if="sessions === undefined">
+      <!-- Handle loading state -->
+    </div>
+
+    <div v-else>
+      <p>Your current session expires at {{ sessions[0].expireAt }}.</p>
+    </div>
+  </template>
+  ```
```

Comment:
I'd change the example here, `sessionList` is ideal for building a UserButton for multi session apps, maybe we can showcase that